images/tier1/blog.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18" mirror-rtl="true">
```

Comment:
omgsvg doesn't remove it, and neither does svgo (which makes sense since they are the same), i'll manually remove it cause it doesn't seem to be doing much